Src/Base/AMReX_GpuLaunchFunctsC.H

```diff
@@ -175,6 +199,31 @@ void ParallelFor (T n, L&& f) noexcept
     ParallelFor(n, std::forward<L>(f));
 }
 
+/** ParallelFor with a SIMD Width (in elements)
+ *
+ * SIMD load/Write-back operations need to be performed before/after calling this.
+ *
+ * @tparam WIDTH SIMD width in elements
+ * @tparam N index type (integer)
+ * @tparam L function/functor to call per SIMD set of elements
+ */
+template <int WIDTH, typename N, typename L, typename M=std::enable_if_t<std::is_integral_v<N>> >
+AMREX_ATTRIBUTE_FLATTEN_FOR
+void ParallelForSIMD (N n, L const& f) noexcept
+{
+    N i = 0;
+    // vectorize full lanes
+    for (; i + WIDTH < n; i+=WIDTH) {
+        f.template operator()<WIDTH>(SIMDindex<WIDTH, N>{i});
```

Comment:
```suggestion
       f(SIMDindex<WIDTH, N>{i});
```
Needs to be like this if lambdas should work. Note that this requires changing the simd operator() in impactx.

Reply:
Ok, easy change. Testing and pushing in a second.

Reply:
fixed here and in ImpactX :)